src/ML-METATOMIC/pair_metatomic.cpp

```diff
@@ -178,6 +163,61 @@ void PairMetatomic::settings(int argc, char ** argv) {
     // load the model and get it's capabilities (including supported devices)
     mta_data->load_model(this->lmp, model_path, extensions_directory);
 
+    // Check that the model has the required outputs
+    const auto& outputs = mta_data->capabilities->outputs();
+    auto energy_output = outputs.find("energy");
+    if (energy_output == outputs.end()) {
+        lmp->error->all(FLERR,
+            "the model at '{}' does not have an 'energy' output, "
+            "we can not use it in pair_style metatomic",
+            model_path
+        );
+    }
+
+    mta_data->energy_output_per_atom = energy_output->value()->per_atom;
+    mta_data->energy_output = torch::make_intrusive<metatomic_torch::ModelOutputHolder>();
+    mta_data->energy_output->explicit_gradients = {};
+    mta_data->energy_output->set_quantity("energy");
+    mta_data->energy_output->set_unit(this->energy_unit);
+    mta_data->energy_output->per_atom = false;
```

Comment:
I don't understand the logic of `per_atom` vs non-`per_atom` energy requests here. Can we have a comment?